sidekick/models.py

```diff
@@ -18,3 +22,9 @@ def __str__(self):
 
     def get_rapidpro_client(self):
         return TembaClient(self.url, self.token)
+
+
+@receiver(post_save, sender=User)
+def user_token_creation(sender, instance, created, **kwargs):
+    if created:
+        Token.objects.create(user=instance)
```

Comment:
@DevChima I think this is because I forgot to test this IF statement, which I added a test for and then rebased on. I suspect that this is because Travis -> CodeCov didn't update. Let me re-run the travis checks and see if it updates.

Reply:
Nope! Looks like this was my fault. The new test was calling the function that it was supposed to. Pushing change to test now.